src/sage/structure/sequence.py

```diff
@@ -212,6 +212,11 @@ def Sequence(x, universe=None, check=True, immutable=False, cr=False, cr_str=Non
         Traceback (most recent call last):
         ...
         TypeError: unable to convert a to an element of Integer Ring
+
+    This used to construct ``[]`` since the iterator is consumed twice::
```

Comment:
```suggestion
    The iterator used to be consumed twice during the construction when
    there is no coercion (see :issue:`37358`)::
```
I think this helps clarify what the doctest is protecting against, but there likely is a better way to explain...